darts/timeseries.py

```diff
@@ -880,6 +880,41 @@ def is_within_range(self,
         index = self.time_index()
         return index[0] <= ts <= index[-1]
 
+
+    def map(self,
+            fn: Callable[[float], float],
+            cols: Optional[Union[List[int], int]] = None) -> 'TimeSeries':
+        #TODO check if typing is correct
+        """
+        Applies the function fn to all values in this TimeSeries, or, to only those
```

Comment:
You can write `Applies the function fn elementwise to all values`.

Reply:
Done. (Specifically, added word 'elementwise')